shop/src/main/java/ro/msg/learning/shop/dto/ProductDto.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class ProductDto {
+    private UUID productId;
```

Comment:
Maybe it will be better to use for DTOs string instead of UUID because UUID is not a standard type to use in a JSON object. Then in the mappers we will transform this string into UUID and use it in database actions.